### DISCUSSION TITLE
[Feature Request] Use timestamp of first button press for bookmarks

### DISCUSSION THREAD

**Discussion by kencano** [PaulWoitaschek/Voice#1413] [Ideas]
When I start creating a bookmark while continuing to listen to the audiobook the timestamp of the bookmark will later be the point when I pressed "OK" after entering the description. In order to be able to leave quick bookmarks without having to pause the audiobook I'd like to recommend using the timestamp of the first button press (bookmark icon) instead of the last button press (create bookmark "OK"). 

App-Version: 4.3.2
Phone Name: Google Pixel 2
Android Version: 10 (QQ1A.200205.002)

**Comment by voidplayer** [comment]
Yeah, you want the bookmark when you pressed the bookmark key

It makes no sense to make the bookmark point to when you finished typing the label
